scripts/go-test-all.py

```
@@ -28,12 +28,25 @@ def find_go_modules(directory):
 def run_tests_for_module(module, *runargs):
     """ Run the unit tests for the given module. """
     os.chdir(module)
-    
+
     print(f"Running unit tests for {module}")
 
     # add runargs to test_command
     test_command = f'go test -mod=readonly {" ".join(runargs)} ./...'
-    result = subprocess.run(test_command, shell=True)
+    result = subprocess.run(test_command, shell=True, capture_output=True, text=True)
```

Ruff (0.15.9)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)